locales/en-US/messages.properties

```diff
@@ -123,6 +123,7 @@ prjListDeleteProjectBtn=Delete
 ## Shared ##
 ############
 
+locale=en_US
```

Comment:
coz it needs to be rendered into the requirejs config like out here https://github.com/mozilla/thimble.webmaker.org/pull/1386/files#diff-270f212d36b8a2447fd4f97c986a0b06R4
I was thinking about other ways to do this but I can't come up with one :( (suggestions welcome)
But I think this will be a bit clearer when I add the client strings into this file